cc/defs.bzl

```diff
@@ -99,6 +99,22 @@ def _create_hdrs(**kwargs):
         visibility = kwargs.get("visibility", ["//visibility:private"]),
     )
 
+def _symbolizer_env(val):
+    return select({
+        # The + operator is not supported on dict and select types so we need to be
+        # clever here.
+        Label("//cc:_enable_symbolizer_x86_64_linux"): dict(val, **{"ASAN_SYMBOLIZER_PATH": "$(location @x86_64-linux-llvm//:symbolizer)"}),
+        Label("//cc:_enable_symbolizer_x86_64_darwin"): dict(val, **{"ASAN_SYMBOLIZER_PATH": "$(location @x86_64-darwin-llvm//:symbolizer)"}),
+        "//conditions:default": {},
+    })
+
+def _symbolizer_data():
+    return select({
+        Label("//cc:_enable_symbolizer_x86_64_linux"): ["@x86_64-linux-llvm//:symbolizer"],
+        Label("//cc:_enable_symbolizer_x86_64_darwin"): ["@x86_64-darwin-llvm//:symbolizer"],
```

Comment:
Do we need a setting for aarch64?

Reply:
We don't.

We built the llvm distribution for apple silicon from source since there were no pre-built binaries available. I couldn't figure out how to build the extra clang tools (clang-tidy, llvm-symbolizer, etc..)

So this is currently only supported on intel mac and linux.

Reply:
:+1: